isthmus/src/test/java/io/substrait/isthmus/TpchQueryNoValidation.java

```diff
@@ -9,7 +9,7 @@ public class TpchQueryNoValidation extends PlanTestBase {
 
   @ParameterizedTest
   // @ValueSource(ints = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22})
-  @ValueSource(ints = {1, 2, 3, 4, 5, 6, 7, 10, 11, 14, 15, 17, 18, 19, 20, 21})
+  @ValueSource(ints = {1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 14, 15, 17, 18, 19, 20, 21})
```

Comment:
Maybe 22 will pass to?

Reply:
22 (and 8, 9, 13) pass in the main branch.

Reply:
Great! Looks like 16 passes too, so all of them pass now. Let's add all those in then.